examples/deployment.yaml

```diff
@@ -15,6 +15,7 @@
           -  name: kube-monkey
              command:
                - "/kube-monkey"
+             args: ["-v=5", "-log_dir=/path/to/custom/log"]
```

Comment:
One last nit-pick :)

Can we make this ` "-log_dir=/var/log/kube-monkey"`.

I want to make sure if someone grabs the example and tries to run it as is, they have some sane defaults.